cleanlab/object_detection/summary.py

```diff
@@ -34,6 +43,412 @@
+
+
+    Returns
+    -------
+    sorted_idxs: List[Tuple[int, int]], List[Tuple[int, int]]
+        A tuple containing two lists. The first is an array of shape ``(N,)`` containing the number of annotated objects for each image in the dataset.
+        The second is an array of shape ``(N,)`` containing the number of predicted objects for each image in the dataset.
+    """
+    lab_count, pred_count = object_counts_per_image(labels, predictions)
+    lab_grouped = zip([i for i in range(len(labels))], lab_count)
+    pred_grouped = zip([i for i in range(len(predictions))], pred_count)
+
+    sorted_lab = sorted(lab_grouped, key=lambda x: x[1], reverse=True)
+    sorted_pred = sorted(pred_grouped, key=lambda x: x[1], reverse=True)
+
+    return sorted_lab, sorted_pred
+
+
+def plot_class_size_distributions(
+    labels, predictions, class_names=None, class_to_show=MAX_CLASS_TO_SHOW
+):
+    """
+    Plots the size distributions for bounding boxes for each class.
+
+    This plot can help you find annotated/predicted boxes for a particular class that are abnormally big/small.
+
+    Parameters
+    ----------
+    labels:
+        Annotated boxes and class labels in the original dataset, which may contain some errors.
+        Refer to documentation for this argument in :py:func:`object_counts_per_image <cleanlab.object_detection.summary.object_counts_per_image>` for further details.
+
+    predictions:
+        Predictions output by a trained object detection model.
+        Refer to documentation for this argument in :py:func:`object_counts_per_image <cleanlab.object_detection.summary.object_counts_per_image>` for further details.
+
+    class_names: optional
+        Optional dictionary mapping one-hot-encoded class labels back to their original class names in the format ``{"integer-label": "original-class-name"}``.
+
+    class_to_show: optional
+        The number of classes to show in the plots. Classes over `class_to_show` are hidden.
+    """
+    try:
+        import matplotlib.pyplot as plt
+    except ImportError as e:
+        raise ImportError(
+            "This functionality requires matplotlib. Install it via: `pip install matplotlib`"
+        )
+
+    lab_boxes, pred_boxes = bounding_box_size_distribution(
+        labels, predictions, class_names=class_names
+    )
+
+    for i, c in enumerate(lab_boxes.keys()):
+        if i >= class_to_show:
+            break
+        fig, axs = plt.subplots(1, 2, figsize=(10, 5))
+        fig.suptitle(f"Size distributions for bounding box for class {c}")
+        for i, l in enumerate([lab_boxes, pred_boxes]):
+            axs[i].hist(l[c], bins="auto")
+            axs[i].set_xlabel("box area (pixels)")
+            axs[i].set_ylabel("count")
+            axs[i].set_title("annotated" if i == 0 else "predicted")
+
+        plt.show()
+
+
+def plot_class_distribution(labels, predictions, class_names=None):
```

Comment:
Can we make class_names take in a limited dictionary and only plot those classes? For example if data has classes 0,1,2 which are car,person,cup respectively if you pass in `class_names={0: car}` then plot_class_distributions will only plot the car class. This way you can use this function to visualize a specific class distribution, not all distributions every time. 

Remember to change docstring definition :D